cves/CVE-2017-14056.yml

```diff
@@ -306,8 +324,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: Should not trust the rl2 header to be correct. 
```

Comment:
Correct, I'd just add some more padding to the front of the sentence, such as "the function should not..." to make this read better.